### DISCUSSION TITLE
Bug

### DISCUSSION THREAD

**Discussion by Zaenalos** [community/community#125027] [Mobile] [answered]
### Select Topic Area

Bug

### Body

I'm having a bug animation in my profile section/tab since i updated the GitHub app.

https://github.com/community/community/assets/77824453/1d24d4e0-87dc-4d9f-8228-51ad2be0cc76

**Comment by alcere** [comment, marked as answer]
Hi @Zaenalos! We apologize for any inconvenience caused! Could you please confirm whether you are using the beta version of GitHub Android on your device? Thanks!

**Comment by Zaenalos** [comment]
Yes sir

**Comment by Zaenalos** [comment]
Okay sir, the bug stopped after i installed the Non beta version. Thank you
for your support.

**Reply by alcere** [reply]
We've rolled out some enhancements to the Profile Readme that are currently only available to Beta users, and it seems that this update has unfortunately caused some issues. We apologize for this inconvenience! Your help in reporting this issue is greatly appreciated, and we will work to fix the problem as quickly as possible. Please keep your feedback coming in the future!

**Reply by Zaenalos** [reply]
Thank you! I'll give a feedback soon after I install the new update of BETA.

**Reply by alcere** [reply]
We have just released an Beta update 1.164.0 that should resolve the issue you mentioned above. Would you mind giving it a try, please? Thanks!

**Reply by Zaenalos** [reply]
I just downloaded the new update and everything is working fine now! Thank you so much! 😁


![Screenshot_2024-05-29-07-38-10-821_com.android.vending.jpg](https://github.com/community/community/assets/77824453/a81bd874-3cdd-4580-b68b-dd68528d5b21)

![Screenshot_2024-05-29-07-38-28-183_com.github.android.jpg](https://github.com/community/community/assets/77824453/78733083-6f71-4fc5-a63c-1daa4b8efe66)

**Reply by alcere** [reply]
Awesome! Thanks for sharing!
